### PR TITLE
UP-4686: Add new option to set Maven thread value for build. 

### PR DESCRIPTION
**Pull request by bjagg**
Updated antHelp.txt in docs/

https://issues.jasig.org/browse/UP-4686

### CODE REVIEW

**Comment by jameswennmacher** [comment]
Based on your comments in about this setting (see https://groups.google.com/a/apereo.org/forum/#!topic/uportal-dev/GtcMaZn4D9c), could we instead default it to 2 rather than being not defaulted?

**Comment by jameswennmacher** [comment]
Also please update the UP4.3 manual to mention the option of setting this value using MVN_OPTS.

**Comment by drewwills** [comment]
I support James' suggestion, unless someone can point out a downside.

**Comment by bjagg** [comment]
Waiting for some feedback on a good default. Someone at Oakland experimented and found 1.5 worked well for a default value.

I looked at MAVEN_OPTS. I could not get it to work with this parameter. Some searching exposed that only JVM parameters are allowed, unfortunately.

File: build.xml [hunk 1]
```diff
@@ -1683,6 +1683,11 @@ IMPORTANT: The '-Dpattern=' property is no longer a regular expression and is a
         <property name="goal8" value="-Djasig.ignore" />
         <property name="goal9" value="-Djasig.ignore" />
 
+        <condition property="threads" value="-T ${maven.threads.per.cpu}C">
```
Comment (jameswennmacher) [inline build.xml]: Propose changing this to
```
<condition property="threads" value="-T ${maven.threads.per.cpu}C" else="-T 2C">
```

Comment (bjagg) [inline build.xml]: I received feedback that 1.5 was an optimal value from Oakland. It really depends on he server details.

Great idea! Using `else=` will make that section shorter and clear.

**Comment by jameswennmacher** [comment]
+1 on merge

**Pull request merged**